python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, uri):
             for i in range(3):
                 (ok, username, password) = QgsCredentials.instance().get(conninfo, username, password, err)
                 if not ok:
-                    raise ConnectionError(e)
+                    raise ConnectionError(QCoreApplication.translate('db_manager', 'Could not connect to database as user {}').format(user=username))
```

Comment:
```suggestion
                    raise ConnectionError(QCoreApplication.translate('db_manager', 'Could not connect to database as user {user}').format(user=username))
```

I think you can't use a dict without using the key in the sentence

Reply:
Haha, thanks